test/config/300-certificate.yaml

```diff
@@ -0,0 +1,43 @@
+# Copyright 2019 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
```

Comment:
Not a blocker for this PR, but I think those yaml files could be shared across net-http01 and net-certmanager. Wondering if we should put them into a sharable places?

Reply:
Yeah we have the same problem for kingress resources, so maybe we should move these to `knative.dev/networking/test/{kcert,kingress}/config` when we create that?  We should probably aim for 0.15 (cc @tcnghia )